web/handlers.go

```diff
@@ -176,13 +176,13 @@ func (h Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 		// Add unsafe-eval to the content security policy for faster source maps in development mode
 		devCSP := ""
-		if model.BuildNumber == "dev" {
+		if *c.App.Config().ServiceSettings.EnableDeveloper {
 			devCSP += " 'unsafe-eval'"
 		}
 
 		// Add unsafe-inline to unlock extensions like React & Redux DevTools in Firefox
 		// see https://github.com/reduxjs/redux-devtools/issues/380
-		if model.BuildNumber == "dev" {
+		if *c.App.Config().ServiceSettings.EnableDeveloper {
```

Comment:
That makes sense; I might still maintain the two separate string concatenation operations rather than combining those, so that the comment blocks are associated with each of the individual directives.

Reply:
...but will combine the `if` blocks (just to clarify the plan).

Reply:
I think we can combine the comments as well given they clearly specify what they refer to.

Reply:
Mostly done; I'm still a little uncertain about combining the comments & concatenations because it feels just possible-enough that one of these directives could be removed in future without the other one being affected.  But if you think we should definitely combine those, happy to push another commit with that change.

Reply:
No worries, it's fine as is. Thanks for taking care of the rest :)

Reply:
Ok, thanks :)